.ado/publish.yml

```diff
@@ -28,6 +28,11 @@ jobs:
         submodules: recursive # set to 'true' for a single level of submodules or 'recursive' to get submodules of submodules
         persistCredentials: true # set to 'true' to leave the OAuth token in the Git config after the initial fetch
 
+      - task: CmdLine@2
```

Comment:
This new task is applying the patch, but at the end of this yml script we publish the npm to npmjs.org as 'react-native-macos'.    As it is, the 'react-native-macos' npm payload will contain the results of the patch.   We don't want that.   

I think we're going to have to split publish into two seperate build pipeline slices: public and office.    The public publish needs the "Bump package version" task to run and then "Prepare package.json for npm publishing as react-native-macos" followed by "Publish react-native-macos to npmjs.org".   But the other steps like "Do Publish" have to be done *before* "Prepare package.json for npm publishing as react-native-macos", hence I think we're going to have to split this file into two and have two parallel publish slices.   @acoates-ms : What do you think?

Reply:
Similarly, the Apple PR steps need to be split.   We really need the PR validation to run as it does now, plus another slice that runs PR validation with the patch applied.